### DISCUSSION TITLE
Downloading Source Code

### DISCUSSION THREAD

**Discussion by dipanbeas** [shogo4405/HaishinKit.kt#224] [Q&A]
Hello,

I can not able to download source code from github. Please help me to download the code for android.

Thanks

**Comment by shogo4405** [comment]
I understand the phrase "unable to download," but I'm not sure specifically what it refers to. Is it that you can't use the git command? Without a bit more detail, I can't provide guidance.

**Comment by dipanbeas** [comment]
The thing is that after downloading the zip file it can not able to unzip and said "windows  can not open the folder". I also attach a screen shot please check.
![imgpsh_fullsize_anim](https://github.com/shogo4405/HaishinKit.kt/assets/61725882/510fe6f7-fe8e-417a-88ee-95195b15d473)

**Reply by shogo4405** [reply]
Maybe it's because of "HaishinKit.kt-main.zip." Renaming it to something like "HaishinKit-main.zip" might work. I'm not familiar with Windows, so I don't know beyond this.

**Comment by dipanbeas** [comment]
I faced same problem when use "HaishinKit-main.zip". Please check the attachment.
![imgpsh_fullsize_anim](https://github.com/shogo4405/HaishinKit.kt/assets/61725882/3833feaf-6b18-4fc1-86c9-ac98e84f742d)

**Reply by shogo4405** [reply]
I don't have Windows, so I'm not familiar with the features provided by GitHub.

**Reply by dipanbeas** [reply]
Can you please share the source code in mail so that I can use it in my project?

**Reply by alexmck** [reply]
@dipanbeas you have .zip twice. Delete the .zip at the end of the filename.

**Reply by dipanbeas** [reply]
Still it not working. Please see the screen shot.

![imgpsh_fullsize_anim](https://github.com/shogo4405/HaishinKit.kt/assets/61725882/dfef1e2f-a78c-4856-887b-dc6c9137a0fb)

**Reply by shogo4405** [reply]
Yes, email support is available starting from $50 per incident. Please make the payment in advance.

**Reply by dipanbeas** [reply]
I can able to unzip the project but when I try to run it then I got the error like "fatal error: 'vulkan/vulkan.hpp' file not found' in android studio.
